odoo/tools/translate.py

```diff
@@ -183,6 +183,15 @@ def translate_attrib_value(node):
 space_pattern = re.compile(r"[\s\uFEFF]*")  # web_editor uses \uFEFF as ZWNBSP
 
 
+def should_force_translatable_element(node):
+    # Some specific nodes (e.g., text highlights) have an auto-updated DOM
+    # structure that makes them impossible to translate. The introduction of a
+    # translation `<span>` in the middle of their hierarchy breaks their
+    # functionalities. The goal here is to force them to be translated as a
+    # whole using the `.o_force_translatable` selector.
+    return "o_force_translatable" in node.attrib.get("class", "").split()
```

Comment:
To be discussed:
I feel the world `translatable` is a terminology used in the translate function code.
If we edit an xml or wrtie js to set `o_force_translatable` to a node, it means the not was not `translatable`
When we say a node is not translatable, usually I think it means every term in the node won't be translated, but not the node won't be translated as a whole.

not sure if we can find a better name
o_translate_inline?

Reply:
Yes, it's a bit misleading, it's actually the same thing for (`TRANSLATED_ELEMENTS` => `INLINE_TRANSLATED_ELEMENTS` ||  `ELEMENTS_TRANSLATED_AS_A_WHOLE`...). `o_translate_inline` seems better and clearer. Code udpated now. Thanks :+1: